src/engraving/tests/environment.cpp

```diff
@@ -51,14 +51,24 @@ static muse::testing::SuiteEnvironment engraving_se(
 
     mu::engraving::loadInstrumentTemplates(":/data/instruments.xml");
 
-    std::shared_ptr<::testing::NiceMock<mu::engraving::EngravingConfigurationMock> > configurator
-        = std::make_shared<::testing::NiceMock<mu::engraving::EngravingConfigurationMock> >();
+    using ECMock = ::testing::NiceMock<mu::engraving::EngravingConfigurationMock>;
+
+    std::shared_ptr<ECMock> configurator(new ECMock(), [](ECMock*) {}); // no delete
     ON_CALL(*configurator, isAccessibleEnabled()).WillByDefault(::testing::Return(false));
     ON_CALL(*configurator, defaultColor()).WillByDefault(::testing::Return(muse::draw::Color::BLACK));
-    mu::engraving::EngravingItem::engravingConfiguration.set(configurator);
+
+    muse::modularity::globalIoc()->unregister<mu::engraving::IEngravingConfiguration>("utests");
```

Comment:
Should that be `"engraving"` instead of `"utests"`? Or how does this work?

Reply:
In fact, this does not matter, this is just information for statistics, the output of which I not did (that is, the idea was to show information about which modules register which interfaces). This is not involved in the work itself.